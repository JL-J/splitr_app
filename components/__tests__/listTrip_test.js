import React from 'react';
import { View, Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import ListTrip from '../listTrip';

describe('<ListTrip />', function() {
  it('renders the component correctly', () => {
    const snap = renderer.create(
      <ListTrip />).toJSON();
    expect(snap).toMatchSnapshot();
  });

  it('renders four text input components', () => {
    const wrapper = shallow(<ListTrip />);
    expect(wrapper.find(Text)).toHaveLength(4);
  });

});
