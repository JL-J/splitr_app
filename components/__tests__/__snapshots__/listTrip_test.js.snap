// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListTrip /> renders the component correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View>
    <Text>
      Name: 
    </Text>
    <Text>
      Location: 
    </Text>
    <Text>
      Start Date: 
    </Text>
    <Text>
      End Date: 
    </Text>
  </View>
</View>
`;
